lms/djangoapps/discussion/rest_api/utils.py

```diff
@@ -351,3 +354,51 @@ def get_archived_topics(filtered_topic_ids: List[str], topics: List[Dict[str, st
             if topic['id'] == topic_id and topic['usage_key'] is not None:
                 archived_topics.append(topic)
     return archived_topics
+
+
+def send_response_notifications(thread, course, creator, parent_id=None):
```

Comment:
Would it not be cleaner to have separate methods for sending each of the response notification? like 
- `send_new_response_notification`
- `send_new_comment_notification`
- `send_new_response_on_comment_notification`

Reply:
Updated , moved to seprate class.